tailwind.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  content: ["./*.{html,js}"],
+  theme: {
+    extend: {
+      boxShadow: {
+        'real': '0 1px 6px 0 #171717',
```

Comment:
use the the name that have the sense of what you're doing `real` to `box-shadow` no one can understand what is it.